src/XliffTasks/XliffTasks.csproj

```diff
@@ -1,7 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.4</TargetFramework>
+    <TargetFrameworks>netcoreapp1.1;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.1</TargetFrameworks>
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
```

Comment:
Dogfooding note: I struggled with both default items and implicit refs in making this change.

* Default items are not present in outer evaluation meaning my content was not going to nupkg.
* Needed to add PrivateAssets to Microsoft.NETCore.App package ref. Could have used "Update", but I'm finding that I much prefer these defaults off and just being able to see all items and refs by reading the project file.